src/main/kotlin/apply/application/MissionService.kt

```diff
@@ -15,19 +17,27 @@ class MissionService(
 ) {
     fun save(request: MissionData) {
         check(evaluationRepository.existsById(request.evaluation.id)) { "존재하지 않는 평가 id 입니다." }
-        check(!missionRepository.existsByEvaluationId(request.evaluation.id)) { "해당 평가에 이미 등록된 과제가 있습니다." }
+        check(!(missionRepository.existsByEvaluationId(request.evaluation.id) && request.id == 0L)) { "해당 평가에 이미 등록된 과제가 있습니다." }
```

Comment:
아래 코드로 변경하면 불필요한 쿼리 생성을 방지할 수 있습니다. (Short circuit evaluation)

```suggestion
        check(!(request.id == 0L && missionRepository.existsByEvaluationId(request.evaluation.id))) { "해당 평가에 이미 등록된 과제가 있습니다." }
```